Makefile

```diff
@@ -65,7 +65,9 @@ bump-major:
 	bumpversion major
 
 release:
-	python setup.py register sdist bdist_wheel upload --sign --identity="$(PGPIDENT)"
+	python -m pip install --upgrade build setuptools wheel twine
```

Comment:
setuptools and wheel are not needed, `build` will create an isolated environment with build dependencies